src/app/users/users-achievements/users-achievements.component.html

```diff
@@ -8,7 +8,10 @@
   <mat-toolbar class="primary-color font-size-1">
     <span>{{user.firstName.concat(' ', user.lastName).trim() || user.name}}</span>
     <span class="toolbar-fill"></span>
-    <a mat-raised-button color="accent" i18n [routerLink]="['/myAchievements/update']" *ngIf="ownAchievements"><mat-icon>mode_edit</mat-icon><span>Edit Achievements</span></a>
+    <a mat-raised-button color="accent" i18n [routerLink]="['/myAchievements/update']"><mat-icon>mode_edit</mat-icon>
+	  <span *ngIf="!ownAchievements">Add Achievements</span>
```

Comment:
I missed this in my review, but the property to test for this is `achievementNotFound` instead of `ownAchievements`.

Also the span tags should be indented two spaces further than the `<a>` tag, and then ending `</a>` tag should be in line with the starting tag.

Reply:
@paulbert I've made the changes.